spec/app/ics-028-cross-chain-validation/system_model_and_properties.md

```diff
@@ -0,0 +1,275 @@
+<!-- omit in toc -->
+# CCV: System Model and Properties
+[&uparrow; Back to main document](./README.md)
+
+<!-- omit in toc -->
+## Outline
+- [Assumptions](#assumptions)
+- [Desired Properties](#desired-properties)
+  - [Validator sets, validator updates and VSCs](#validator-sets-validator-updates-and-vscs)
+  - [Staking Module Interface](#staking-module-interface)
+  - [Validator Set Update](#validator-set-update)
+- [Correctness Reasoning](#correctness-reasoning)
+
+## Assumptions
+[&uparrow; Back to Outline](#outline)
+
+CCV lives between IBC, Cosmos SDK, and the Staking module. 
+In this section we specify what we assume about these other components, 
+i.e., CCV relies on the following assumptions: *Valid Blockchain*, *Correct Relayer*, *Validator Update Provision*, and *Unbonding Safety*. 
+
+Intuitively, CCV safety relies on the *Valid Blockchain* assumption, and CCV liveness relies on the *Correct Relayer* assumption. 
+The *Validator Update Provision* and *Unbonding Safety* assumptions define what is needed from the provider Staking module. 
+A more thorough discussion of the environment in which CCV operates is given in the section [Placing CCV within Cosmos SDK](./technical_specification.md#placing-ccv-within-cosmos-sdk).
+
+- ***Valid Blockchain***: Both the provider and the consumer chains are *valid*. 
+  This means that the protocols are executed correctly and the underlying consensus algorithm satisfies both safety and liveness properties. 
+  For more details, take a look at the [Tendermint Paper](https://arxiv.org/pdf/1807.04938.pdf).
```

Comment:
```suggestion
- ***Safe Blockchain***: Both the provider and the consumer chains are *safe*. This means that, for every chain, the underlying consensus engine satisfies safety (e.g., the chain does not fork) and the execution of the state machine follows the described protocol. 

- ***Live Blockchain***: Both the provider and the consumer chains are *live*. This means that, for every chain, the underlying consensus engine satisfies liveness (i.e., new blocks are eventually added to the chain).

> **Note**: Both *Safe Blockchain* and *Live Blockchain* assumptions require the consensus engine's assumptions to hold, e.g., up to a third of validators can be Byzantine. For more details, take a look at the [Tendermint Paper](https://arxiv.org/pdf/1807.04938.pdf).
```

@josef-widder  do you agree with this re-def of the Valid Blockchain assumption? Clearly it must be made consistent throughout the spec.  

Note that *Live Blockchain* is needed only for liveness, while *Safe Blockchain* is needed for both, since *Correct Relayer* requires some state to be updated in order to make progress and *Correct Relayer* is needed for liveness.